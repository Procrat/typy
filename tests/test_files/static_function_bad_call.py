class A:
    def bestaande_functie():
        pass


A().bestaande_functie()
