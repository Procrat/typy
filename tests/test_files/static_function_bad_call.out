bestaande_functie expected 0 parameters, but received 1
